l10n_it_causali_pagamento/models/causale_pagamento.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class CodiceCarica(models.Model):
+    _name = 'causale.pagamento'
+    _description = 'Causale Pagamento'
+
```

Comment:
data la lunghezza del campo name, in alcuni casi l'interfaccia di Odoo va fuori fase spostandosi da un record all'altro, per cui aggiungerei un display_name
    _rec_name = 'display_name'